app/models/variant_override.rb

```diff
@@ -15,6 +15,7 @@ class VariantOverride < ApplicationRecord
   validates :variant, presence: true
   # Default stock can be nil, indicating stock should not be reset or zero, meaning reset to zero. Need to ensure this can be set by the user.
   validates :default_stock, numericality: { greater_than_or_equal_to: 0 }, allow_nil: true
+  validates :price, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
I think in the case of Variant Overrides, both `count_on_hand` and `price` can be nil. There's a handy chart here:

https://github.com/openfoodfoundation/openfoodnetwork/pull/7699/files#diff-c8114f90762e3aa37ef076badebf5145cd269fc75ac7d1665b45016f48233811L3-R16

The `nil` values imply that attribute is not being overridden.

Reply:
We could use `allow_nil: true` as in the line above this one.